use_case_guide/real_time_dashboards.rst

```diff
@@ -21,14 +21,6 @@ volume is also high, and both historical and live data are important.
 In this section we'll demonstrate how to build part of the first example, but this
 architecture would work equally well for the second and many other use-cases.
 
-Running It Yourself
--------------------
-
-There will be code snippets in this tutorial but they don't specify a complete system.
```

Comment:
Now that we aren't doing complicated stuff in the shell I removed the link to the github repo. The guide seems pretty self-contained now.